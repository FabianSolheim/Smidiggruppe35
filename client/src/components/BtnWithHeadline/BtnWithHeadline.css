.headlineText {
    font-size: 30px;
    text-align: center;
    margin-bottom: 20px;
}
.buttonUnderHeadline {
    font-family: "Montserrat", sans-serif;
    border-radius: 0!important;
    height: 30px;
    width: 120px;
    background-color: #034F7A;
    color: white;
    border: 0;
    font-size: 16px;
    display:block; margin: 0 auto
}