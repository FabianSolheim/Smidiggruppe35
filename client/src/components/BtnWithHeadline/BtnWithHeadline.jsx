import React from "react";
import "./BtnWithHeadline.css";

export function BtnWithHeadline(headlineText, btnText) {
  return (
    <div>
      <h1 className="headlineText">{headlineText}</h1>
      <button className="buttonUnderHeadline">{btnText}</button>
    </div>
  );
}
