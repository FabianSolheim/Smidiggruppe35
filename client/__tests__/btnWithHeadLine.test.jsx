import ReactDOM from "react-dom";
import React from "react";
import { BtnWithHeadline } from "../src/components/BtnWithHeadline/BtnWithHeadline";

describe("btnWithHeadline test", () => {
  it("shows btnWithHeadline component", () => {
    const domElement = document.createElement("div");
    ReactDOM.render(<div>{BtnWithHeadline}</div>, domElement);
    expect(domElement.innerHTML).toMatchSnapshot();
  });
});
