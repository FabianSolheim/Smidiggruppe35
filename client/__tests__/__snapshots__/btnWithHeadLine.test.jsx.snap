// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`btnWithHeadline test shows btnWithHeadline component 1`] = `"<div></div>"`;
